lib/src/biometric_storage.dart

```diff
@@ -120,6 +126,36 @@ class AndroidPromptInfo {
       };
 }
 
+/// iOS specific configuration of the prompt displayed for biometry.
+class IOSPromptInfo {
```

Comment:
I think i would prefer having it called `IosPromptInfo` in dart - i think that's the [preferred capitalisation convention within flutter] 🤔️(https://github.com/flutter/plugins/blob/09ec5192f3ce31622631458ba9655cc257a07951/packages/device_info/device_info_platform_interface/lib/model/ios_device_info.dart#L8). (also like `Uri`, etc. - although i'm not sure why its `Platform.iOS` 😅️

Reply:
Okay changed 😄 , seems to me that dart is full of these inconsistencies, but which language isn't 😄